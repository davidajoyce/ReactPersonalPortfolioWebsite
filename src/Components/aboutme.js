import React, { Component } from 'react';
import { Grid, Cell } from 'react-mdl';

class Aboutme extends Component{
  render(){
      return(
        <div style={{width: '100%', margin: 'auto'}}>
          <Grid className="landing-grid">
            <Cell col={12}>
            </Cell>
            <Cell className="about-me-cell" col={12}>
                <div className="banner-text">
                  <h1>About Me</h1>

                <hr/>
                <h3>Software engineer using Java and C# to develop a Java Enterprise utility billing application for Gentrack Ltd along with developing APIs using the Java Spring framework.
          In my spare time, I love trying out new technologies, for instance, I develop python scripts for my familyâ€™s supermarket business to obtain key data insights. Another project I've developed in my spare time is a Java Spring Web feedback section using Spring MVC and MySQL.</h3>
                </div>
            </Cell>
          </Grid>
        </div>
      )
    }
}

export default Aboutme;
