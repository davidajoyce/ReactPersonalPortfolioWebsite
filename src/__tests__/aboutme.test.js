import React from 'react'
import aboutme from '../Components/aboutme'
import { shallow } from 'enzyme';

describe('aboutme', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<aboutme />);
    expect(wrapper).toMatchSnapshot();
    // On the first run of this test, Jest will generate a snapshot file automatically.
  });
});