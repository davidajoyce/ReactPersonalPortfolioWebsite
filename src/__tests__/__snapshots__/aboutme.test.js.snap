// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`aboutme renders correctly 1`] = `ShallowWrapper {}`;
